src/documents/SwitchAPI.md

```diff
@@ -0,0 +1,224 @@
+            <td>
+                An object containing any key/value pairs representing valid HTML attributes and their respective values. In particular, an integrator should define a key/value pairing for <code>"aria-label"</code> or <code>"aria-labelledby"</code>.
+                <div class="infusion-docs-note"><strong>Note:</strong> If the switch controls another element on the page, <code>"aria-controls"</code> should be added to indicate the relationship.</div>
+            </td>
+            <td>
+                <pre>
+                    <code>
+attrs: {
+    role: "switch",
+    tabindex: 0
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td><code>model</code></td>
+            <td>The state of the switch.</td>
+            <td><code>true</code> is "on", <code>false</code> is off</td>
+            <td>
+                <pre>
+                    <code>
+model: {
+    enabled: false
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Selectors ###
+
+The `selectors` option is an object containing CSS-based selectors for the various parts of the Switch. Supported selectors are:
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Description</th>
+            <th>Default</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>on</code></td>
+            <td>Identifies the element for the "on" identifier</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    on: ".flc-switchUI-on"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+
+        <tr>
+            <td><code>off</code></td>
+            <td>Identifies the element for the "off" identifier</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    off: ".flc-switchUI-off"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+        <tr>
+            <td><code>selectables</code></td>
+            <td>Identifies the DOM element that represents the actual UI switch control.</td>
+            <td>
+                <pre>
+                    <code>
+selectors: {
+    control: ".flc-switchUI-control"
+}
+                    </code>
+                </pre>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Styling the Switch
+
+The Switch includes default CSS styles that it applies to its elements. The application of styles is based on known class names. The _default_ class names are described below, and can be used by including the Switch stylesheet:
+
+```html
+<link href="components/switch/css/Switch.css" type="text/css" rel="stylesheet" media="all">;
+```
+
+<div class="infusion-docs-note"><strong>Note:</strong> Switch.css is generated from the Switch.styl Stylus file.</div>
```

Comment:
Would be helpful to add a link or include instructions of how to compile stylus into css.